package.json

```diff
@@ -10,10 +10,26 @@
     "start": "next start"
   },
   "dependencies": {
+    "firebase-admin": "^8.6.0",
```

Comment:
clean up firebase packages by running npm uninstall package-name e.g. npm uninstall firebase-admin firebase-functions -> like this with space separation

Reply:
done.